src/Main.java

```diff
@@ -1,5 +1,223 @@
-        System.out.println("Hello world!");
+
+        Scanner reader = new Scanner(System.in);
+        // Task 1
+
+        System.out.println("Task 1: ");
+
+        System.out.println("Please enter the length of array you want to define: ");
+        int arrayDesiredLength = reader.nextInt();
+        int[] integerArray = new int[arrayDesiredLength];
+        System.out.println("Length of array " + integerArray.length);
+
+        for (int i = 0; i <= integerArray.length - 1; i++) {
+            System.out.println("Please enter the " + (i + 1) + ". element value: ");
+            int value = reader.nextInt();
+            integerArray[i] = value;
+        }
+
+        for (int i = (integerArray.length - 1); i >= 0; i--) {
+            System.out.println(integerArray[i]);
+            if ((integerArray[i] % 2 == 0)) {
+                System.out.println("The number is even number.");
+            } else {
+                System.out.println("The number is not an even number.");
+            }
+            if ((integerArray[i] > 10)) {
+                System.out.println("The number is greater than 10.");
+            } else {
+                System.out.println("The number is not greater than 10.");
+            }
+        }
+
+        // Zadatak 2
+
+        System.out.println("Task 2: ");
+        int[] arrayOFNumbers = {123, 23, 3, 0, 1, 2, 4, 5, 120, 19, 20, 21, 22, 23, 24, 26, 98};
+        String[] ordinalNumbers = {"First", "Second", "Third", "Fourth", "Fifth", "Sixth", "Seventh", "Eighth", "Ninth", "Tenth", "Eleventh", "Twelfth", "Thirteenth", "Fourteenth", "Fifteenth"};
+        int condition;
+
+        if (arrayOFNumbers.length <= ordinalNumbers.length) {
+            condition = arrayOFNumbers.length;
+        } else {
+            condition = ordinalNumbers.length;
+        }
+
+        for (int i = 0; i <= (condition - 1); i++) {
+            System.out.println(ordinalNumbers[i] + " number in array: " + arrayOFNumbers[i]);
+        }
+
+        // Zadatak 3
+
+        System.out.println("Task 3: ");
+
+        int[] anyArray = {1, 2, 3, 4, 5, 6, 7};
+        System.out.println(anyArray.length);
+        int[] anyArrayToOpposite = new int[anyArray.length];
+        for (int i = (anyArray.length - 1); i >= 0; i--) {
+            anyArrayToOpposite[(anyArray.length - 1) - i] = anyArray[i];
+        }
+
+        System.out.println("New opposite array is : ");
+        for (int i = 0; i <= (anyArrayToOpposite.length - 1); i++) {
+            System.out.println(anyArrayToOpposite[i]);
+        }
+
+        // Zadatak 4
+
+        System.out.println("Task 4: ");
+
+        System.out.println("Please insert a word to check if it is palindrome:");
+        String wordPalindrome = reader.next();
+
+        char[] stringToChar = wordPalindrome.toCharArray();
+        int middleOfArray = (stringToChar.length / 2) - 1;
+        int toCheckPalindrome = 0;
+
+        for (int i = 0; i <= middleOfArray; i++) {
```

Comment:
OR you could use boolean ifPalindrome =true
and if the match does not happen it sets it to false no need to validate lengths no more.